services/m3aggregator/config/aggregator.go

```diff
@@ -369,7 +371,7 @@ func (c *streamConfiguration) NewStreamOptions(instrumentOpts instrument.Options
 }
 
 type placementManagerConfiguration struct {
-	KVConfig         kv.Configuration               `yaml:"kvConfig"`
+	KVConfig         kv.OverrideConfiguration       `yaml:"kvConfig"`
```

Comment:
Do we need to set loggers here as well?

Reply:
Nope, I think normal logger here is good enough, for flush times mgr we need to config to warn level logger